routes/api.php

```diff
@@ -13,6 +13,6 @@
 |
 */
 
-Route::middleware('auth:api')->get('/user', function (Request $request) {
-    return $request->user();
-});
+// Route::middleware('auth:api')->get('/user', function (Request $request) {
```

Comment:
Can we delete anything we're not using instead of commenting it out?

Reply:
Even the Laravel examples?

Reply:
I believe its useful to have one example handy for each kind of route. Unless you feel very strongly, I've leaving these comments in for now.